flask.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Flask
+layout: 2017/sheet
+category: Flask  # could be split later on
+updated: 2018-04-06
+prism_languages: [python, bash, django]
+intro: |
+  Flask is a microframework for Python based on Werkzeug, Jinja 2 and good intentions.
+tags:
+  - WIP
+  - Python
+---
+
+{% raw %}
+
+Flask
+-----
+
+## Configuration
+
+### Setting Configuration
+
+```python
+app = Flask(__name__)
+app.config['DEBUG'] = True
+app.config.from_object('yourapplication.default_settings')
+app.config.from_envvar('YOURAPPLICATION_SETTINGS')
+```
+
+### Builtin Configuration Values
```

Comment:
I'd keep this in "Sentence case" rather than "Title Case" - just to be consistent with the rest of the site :)

Reply:
Sure thing! I was mostly copy-pasting stuff from the documentation haha